package.json

```
@@ -27,8 +27,21 @@
         "fetch": "npm-run-all --parallel fetch:*",
         "start": "npm-run-all build:sass --parallel watch:*",
         "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
-        "lint": "eslint --ext=.js,.jsx .",
-        "fix": "npm run lint -- --fix"
+        "minify:svg": "svgo -r -f ./",
+        "lint:js": "eslint --ext=.js,.jsx .",
+        "lint:js:fix": "npm run lint:js -- --fix",
+        "lint:scss": "stylelint '**/*.scss'",
+        "lint:scss:fix": "npm run lint:scss --fix",
+        "lint": "npm-run-all --parallel lint:js lint:scss",
+        "fix": "npm-run-all --parallel lint:js:fix lint:scss:fix"
```

```suggestion
        "fix": "npm-run-all --parallel lint:*:*"
```

I think we can do it this way? Can you once verify if this works?